Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -1090,13 +1090,15 @@ WarpXOpenPMDPlot::WriteOpenPMDFieldsAll ( //const std::string& filename,
           // assume fields are scalar unless they match the following match of known vector fields
           std::string field_name = varname;
           std::string comp_name = openPMD::MeshRecordComponent::SCALAR;
+          std::cout << "field_name=" << field_name << std::endl;
```

Comment:
are you sure you want this and the next couts?

Reply:
oh no, needs to be removed (debug output)